### PR TITLE
bugfix

### PR DESCRIPTION
**Pull request by maximgubar**
- impossibility of files copying/moving

### CODE REVIEW

File: magento-deploy.sh [hunk 1]
```diff
@@ -4,7 +4,7 @@ readarray -t  dirs < .platform-read-write-dirs
 # loop through array(dirs)
 for dir in "${dirs[@]}"
 do
- cp -R "../init/$dir/*" "$dir"|| true
+ cp -R ../init/$dir/* $dir || true
```
Comment (lsmith77) [inline magento-deploy.sh]: are you sure this change is necessary? seems like the quotes should be a good idea to keep just for "safety"

Comment (maximgubar) [inline magento-deploy.sh]: the asterisk symbol is not recognized in this quotes

Comment (pjcdawkins) [inline magento-deploy.sh]: This would be valid, and it would allow for spaces and other special characters in `$dir`:

```bash
"../init/$dir/"*
```

**Comment by philorkill** [comment]
I can confirm that this solution works, the current does not.

**Pull request closed without merging**